mage_ai/frontend/components/PipelineDetail/constants.ts

```diff
@@ -1,4 +1,5 @@
 export const PAGE_NAME_EDIT = 'edit';
+export const MAX_REQ_PRINT_OUTPUT_LINES = 150;
```

Comment:
Is this still hardcoded? Can we get the env var from backend?

Reply:
Updated. We needed to include it in the `statuses` api response, similar to what we do for `DISABLE_NOTEBOOK_EDIT_ACCESS` and `REQUIRE_USER_AUTHENTICATION`.